maven-scm-providers/maven-scm-provider-jazz/src/main/java/org/apache/maven/scm/provider/jazz/JazzScmProvider.java

```diff
@@ -222,7 +222,7 @@ public ScmProviderRepository makeProviderScmRepository( String scmUrl, char deli
             // So we will set port to zero.
             int protocolIndex = jazzUrl.indexOf( ":" ) + 3;     // The +3 accounts for the "://"
             int pathIndex = jazzUrl.indexOf( "/", protocolIndex + 1 );
-            if ( ( protocolIndex != -1 ) && ( pathIndex != -1 ) )
+            if ( pathIndex != -1 )
```

Comment:
Please explain

Reply:
two line above it add 3 to account, so, in the worst case. would be -1 + 3